theano/gradient.py

```diff
@@ -1945,6 +1945,8 @@ class DisconnectedGrad(ViewOp):
     def grad(self, args, g_outs):
         return [disconnected_type() for g_out in g_outs]
 
+    def connection_pattern(self, node):
```

Comment:
Thing is, I'm not exactly certain what to put in the test except than validate that it returns [[False]] which seemed a bit too trivial to be a valuable test. I guess it will at least make sure that the implementation of that method is not altered by accident in the future.